petrinet/examples/sir.json

```diff
@@ -57,7 +61,8 @@
           "R"
         ],
         "properties": {
-          "name": "Recovery"
+          "name": "Recovery",
+          "description": "Recovery interaction of a infected individual"
```

Comment:
Instead of "interaction", we can call this "process".

Reply:
Done, I also removed it for `typed_sir.json` in anticipation!